bindings/rust/generate/generate.sh

```diff
@@ -147,6 +148,8 @@ function create_prefix_headers {
 
 function prepare_crate_dir {
   echo Preparing crate directory: "${CRATE_DIR}"
+  rm -rf "${CRATE_DIR}"
```

Comment:
nit: Probably worth leaving a comment that this will also remove the competition marker (which we want).

Reply:
Fixed in latest commit.